pkg/oidc/oidc.go

```diff
@@ -25,7 +25,13 @@ var (
 		JWKSURL:                "https://token.actions.githubusercontent.com/.well-known/jwks",
 		ExpectedJobWorkflowRef: "kyma-project/test-infra/.github/workflows/image-builder.yml@refs/heads/main",
 	}
-	TrustedOIDCIssuers = map[string]Issuer{GithubOIDCIssuer.IssuerURL: GithubOIDCIssuer}
+	GithubToolsOIDCIssuer = Issuer{
```

Comment:
```suggestion
	GithubToolsSAPOIDCIssuer = Issuer{
```

Reply:
It's minor change, I can introduce it in the next PR for this issue.